src/parser/jobs/pct/modules/StarryMuse.tsx

```diff
@@ -0,0 +1,154 @@
+		super.initialise()
+
+		this.ignoreActions([this.data.actions.STAR_PRISM_CURE.id])
+
+		// Shouldn't also need an Expected GCD Count evaluator since the expected action groups will effectively enforce that
+		this.addEvaluator(new ExpectedActionGroupsEvaluator({
+			expectedActionGroups: [
+				{
+					actions: [this.data.actions.HAMMER_STAMP, this.data.actions.HAMMER_BRUSH, this.data.actions.POLISHING_HAMMER],
+					expectedPerWindow: 3,
+					overrideHeader: <DataLink showName={false} status="HAMMER_TIME" />,
+				},
+				{
+					actions: [this.data.actions.STAR_PRISM],
+					expectedPerWindow: 1,
+				},
+				{
+					actions: [this.data.actions.RAINBOW_DRIP],
+					expectedPerWindow: 1,
+				},
+				{
+					actions: [this.data.actions.COMET_IN_BLACK],
+					expectedPerWindow: 2, // Default to 2, adjust to 1 for the opener
+				},
+				{
+					actions: this.subtractiveActions,
+					expectedPerWindow: 2, // Default to 2, adjust to 3 for the opener
+					overrideHeader: <DataLink showName={false} status="SUBTRACTIVE_PALLETTE" />,
+				},
+				{
+					actions: this.museActions,
+					expectedPerWindow: 1, // Default to 1, we'll assess the window as a triple muse if they actually got more
+					overrideHeader: <DataLink showName={false} action="LIVING_MUSE" />,
+				},
+				{
+					actions: [
+						this.data.actions.MOG_OF_THE_AGES,
+						this.data.actions.RETRIBUTION_OF_THE_MADEEN,
+					],
+					expectedPerWindow: 1,
+				},
+			],
+			suggestionIcon: this.data.actions.STARRY_MUSE.icon,
+			suggestionWindowName: <DataLink showIcon={false} action="STARRY_MUSE" />,
+			suggestionContent: <Trans id="pct.starrymuse.suggestions.expected-actions.content">To maximize your damage, you should make sure your <DataLink status="STARRY_MUSE" /> windows contain all of the recommended actions.</Trans>,
+			severityTiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			adjustCount: this.adjustExpectedActionGroupCounts.bind(this),
+		}))
+
+		this.addEvaluator(new LimitedActionsEvaluator({
+			expectedActions: ADDITIVE_SPELLS.map<TrackedAction>(key => {
+				return {
+					action: this.data.actions[key],
+					expectedPerWindow: 0,
+				}
+			}),
+			suggestionIcon: this.data.actions.FIRE_IN_RED.icon,
+			suggestionWindowName: <DataLink showIcon={false} action="STARRY_MUSE" />,
+			suggestionContent: <Trans id="pct.starrymuse.suggestions.limited-actions.content">Avoid using the additive paint spells during your <DataLink status="STARRY_MUSE" /> windows. The <DataLink status="SUBTRACTIVE_PALLETTE" /> spells have higher potency, allowing you to do more damage during your burst.</Trans>,
+			severityTiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+		}))
+	}
+
+	private adjustExpectedActionGroupCounts(window: HistoryEntry<EvaluatedAction[]>, action: TrackedActionGroup): number {
+		const attemptedTripleMuse = this.countActionsUsed(window, this.museActions) > 1
```

Comment:
Are we straying too far to enforcing a rotation with this?  Is there a more general way to adjust based on the actual number of extra muse actions they used, if they only use a two Muse window?

Reply:
Probably should have noted this in the PR template before, but the expectations were sourced from Eydis' Pictomancer guide on the Balance (https://www.thebalanceffxiv.com/jobs/casters/pictomancer/openers/). There isn't a 2-muse window right now, so having 2 is being treated as "tried and failed to do 3"

Reply:
After further discussion, and watching Eksu's guide video, I did some GCD math, and 2 muse has basically the same GCD expectations as triple muse, so I'm comfortable changing this to "Default 1" for the pre-window prepped muse, then just requiring all motifs painted in the window to get used. Should be fixed now!